dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathEntry.java

```diff
@@ -118,4 +121,11 @@ public static ClassPathEntry of(String coordinates, String filePath) {
 
     return classPath.getAllClasses().stream().map(ClassInfo::getName).collect(toImmutableSet());
```

Comment:
Memo: this `classPath.getAllClasses()` returns empty if `jar` does not exist.
<img width="1358" alt="Screen Shot 2020-04-02 at 11 42 39" src="https://user-images.githubusercontent.com/28604/78269128-27a2e900-74d7-11ea-9dfc-bc3405b8a3f4.png">

Reply:
If we start reading this JAR files without Guava's ClassPath (https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1320 ). I think appropriate behavior is to throw IOException upon nonexistent JAR file.

The existing tests didn't fail for this PR because Guava's ClassPath happened to ignore nonexistent JAR file.